components/recommendation/RecommendationArticle.vue

```diff
@@ -40,13 +39,29 @@ async function copyLink(url: string, event: Event): void {
     </div>
     <div class="media" relative overflow-hidden max-w-120px min-w-120px>
       <img :src="item.image.sizes?.[0]?.url" rounded-lg overflow-hidden w-full ha>
-      <div m-y-4px flex flex-justify-end>
-        <button p-12px text-xl @click="copyLink(item.url, $event)">
-          <div i-ri:share-line />
-        </button>
-      </div>
     </div>
   </NuxtLink>
+  <div flex flex-justify-end p-3 border="b base">
+    <CommonDropdown flex-none ms3 placement="bottom" :eager-mount="command">
```

Comment:
I'm struggling to find information on what `eager-mount` does. I do see that other usages of `CommonDropdown` use it

Reply:
The `CommonDropdown` component renders a package component called `vdropdown` which accepts that prop:
https://floating-vue.starpad.dev/api/#eagermount

Essentially what it does it makes sure the menu is loaded into the DOM before you interact with the button.

Reply:
Though I'm now realizing that `command` is an unused prop so it's defaulting to false anyways, sooooo I'm gonna remove it.

Reply:
Good question!